modules/ResizeChatWindow.js

```diff
@@ -0,0 +1,28 @@
+Modules.ResizeChatWindow = (function () {
+	var bcplus = null;
+
+	var initialize = function(_bcplus) {
+		console.log('Modules.ResizeChatWindow.initialize()');
+		bcplus = _bcplus;
+
+		buildUI();
+		addEventListeners();
+	};
+
+	var buildUI = function() {
+		bcplus.addBoolOption('bcplusResizeChatWindow', 'Chatfenster anpassen', 'bcplusResizeChatWindow', 'Chatfenster', true);
+	};
+
+	var addEventListeners = function() {
+		console.log('Modules.ResizeChatWindow.addEventListeners()');
+		$('#timsChatSmilies').on("click", function() {
+			setTimeout(function() {
+				$(window).resize();
```

Comment:
For the records, always use Window instead of window in userscript context. This goes for global functions like setTimeout as well.